src/core/lv2/Lv2Manager.cpp

```diff
@@ -79,8 +72,7 @@ Lv2Manager::Lv2Manager() :
 	m_supportedFeatureURIs.insert(LV2_URID__unmap);
 	m_supportedFeatureURIs.insert(LV2_OPTIONS__options);
 
-	auto supportOpt = [this](Lv2UridCache::Id id)
-	{
+	auto supportOpt = [this](Lv2UridCache::Id id) {
```

Comment:
Not conforming to our conventions (curly bracket on separate line) ...

Reply:
I'd be okay with using this style. I think a few others are okay with this way as well.

Reply:
OK, I'll resolve this then, as there are no further complaints here.